broker-cli/commands/wallet.js

```diff
@@ -76,22 +80,27 @@ async function balance (args, opts, logger) {
         totalChannelBalance,
         totalPendingChannelBalance,
         uncommittedBalance,
-        uncommittedPendingBalance
+        uncommittedPendingBalance,
+        totalCommitFees
       } = balance
 
       totalChannelBalance = error ? 'Not Available'.yellow : totalChannelBalance.green
       uncommittedBalance = error ? 'Not Available'.yellow : uncommittedBalance
+      totalCommitFees = error ? 'Not Available'.yellow : totalCommitFees
 
       // We fix all pending balances to 8 decimal places due to aesthetics. Since
       // this balance should only be temporary, we do not care as much about precision
       totalPendingChannelBalance = error ? '' : `(${Big(totalPendingChannelBalance).toFixed(8)})`.grey
       uncommittedPendingBalance = error ? '' : `(${Big(uncommittedPendingBalance).toFixed(8)})`.grey
 
-      balancesTable.push([
+      const row = [
         symbol,
         `${totalChannelBalance} ${totalPendingChannelBalance}`,
         `${uncommittedBalance} ${uncommittedPendingBalance}`
-      ])
+      ]
+      if (commitFees) { row.push(`${totalCommitFees}`) }
```

Comment:
display-wise, this might be better as a parenthetical inside of the committed balance